packages/python-packages/api-stub-generator/apistub/nodes/_argtype.py

```diff
@@ -10,15 +10,25 @@
 
 class ArgType:
     """Represents Argument type
-    :param str name: Name of the argument
-    :param str argtype: Type of the argument. for e.g. str, int, BlobBlock
-    :param str default: Default value for the argument, If any
+    :param str name: Name of the argument.
+    :param str argtype: Type of the argument (e.g. str, int, BlobBlock).
+    :param str default: Default value for the argument, If any.
+    :param str keyword: The keyword for the arg type.
+    :param FunctionNode func_node: The function node this belongs to.
     """
-
-    def __init__(self, name, argtype=None, default=None, func_node = None):
+    def __init__(self, name, *, argtype, default, keyword, func_node=None):
         self.argname = name
-        self.argtype = argtype
-        self.default = default
+        if default == inspect._empty or default == inspect.Parameter.empty:
```

Comment:
just curious, how is inspect._empty different from inspect.Parameter.empty?

Reply:
Apparently no difference so I removed the reliance on the private `_empty`